website/docs/r/secret.html.markdown

```diff
@@ -59,24 +59,27 @@ resource "kubernetes_secret" "example" {
     name = "docker-cfg"
   }
 
+  type = "kubernetes.io/dockerconfigjson"
+
   data = {
     ".dockerconfigjson" = jsonencode({
       auths = {
         "${var.registry_server}" = {
-          auth = "${base64encode("${var.registry_username}:${var.registry_password}")}"
+          "username" = var.registry_username,
```

Comment:
Let's keep the usage of a comma at the end of line consistent. I think HCL supports both, we just need to keep to one or the other.

Reply:
Good point! I've looked at other examples and we don't use commas in such cases. Updated. Thank you, @alexsomesan!